fetch.bs

```diff
@@ -4677,11 +4677,26 @@ steps:
 
      <li>
       <p>If <var>response</var>'s <a for=response>timing allow passed flag</a> is not set,
-      then set <var>timingInfo</var> to the result of <a>creating an opaque timing info</a> for
-      <var>timingInfo</var>, set <var>bodyInfo</var> to a new <a for=/>response body info</a>, and
-      set <var>cacheState</var> to the empty string.
+      then:
 
-      <p class=note>This covers the case of <var>response</var> being a <a>network error</a>.
+      <ol>
+       <li>
+        <p>If <var>fetchParams</var>'s
+        <a for="fetch params">request</a>'s <a for=request>mode</a> is "<code>navigate</code>",
+        then abort these steps.
+
+        <p class=note>Reporting timing information for cross-origin navigations without
+        `<code>Timing-Allow-Origin</code>` may expose information about user interaction with that
+        origin.
```

Comment:
Thinking about this again the description here is misleading as it's not just user interaction but general nested document navigations that can be problematic, right?